pkg/operator2/secret.go

```diff
@@ -63,14 +88,19 @@ func newSessionSecretsJSON() []byte {
 }
 
 // needs to be in lib-go
-func randomBits(bits int) []byte {
-	size := bits / 8
-	if bits%8 != 0 {
-		size++
-	}
+func randomBytes(size int) []byte {
 	b := make([]byte, size)
 	if _, err := rand.Read(b); err != nil {
 		panic(err) // rand should never fail
```

Comment:
Bubble up error

Reply:
This is supposed to die.  A failure here means the crypto part of your kernel is broken.  This panic has existed in Openshift since the very beginning with no reported failures (and it would be very obvious if a panic crashed the master).

Reply:
ok : )

Reply:
kept that panic, but return the error from `json.Marshal(secrets)` in `newSessionSecretsJSON()`